src/processor/map/create_factorized_table_scan.cpp

```diff
@@ -83,8 +93,12 @@ std::unique_ptr<PhysicalOperator> PlanMapper::createFTableScanAligned(
     for (auto i = 0u; i < exprs.size(); ++i) {
         colIndices.push_back(i);
     }
+    physical_op_vector_t children;
+    if (child != nullptr) {
```

Comment:
if now ResultCollector can have multiple children, then its child should never be a nullptr. The previous nullptr was for EmptyResultCollector because we force ResultCollector to have exactly one child. Now EmptyResultCollector can just have an empty children vector.